### ISSUE TITLE
`updateFirmware` throws ResponseError error 200 on iOS with Polar 360 when updating from 1.1.10 to 1.1.24

### ISSUE THREAD

**Issue by korzonkiee** [polarofficial/polar-ble-sdk#520]
**Platform on which you observed the bug:** 
- [ ] Android 
- [x] iOS
- [ ] Other
- [ ] Platform is not relevant for this bug

**Device on which you observed the bug:**
- [ ] Polar OH1
- [ ] Polar Verity Sense
- [ ] Polar H10
- [ ] Polar H9
- [ ] Other
- [x] Polar 360
- [ ] Device is not relevant for this bug

**Describe the bug**
Attempting to update firmware from 1.1.10 to 1.1.24 on iOS with Polar 360 always fails with the following error:

```
2024-12-09T06:48:43.803Z I [PC] updateFirmware status: fwUpdateFailed(details: "Error during firmware update: The operation couldn’t be completed. (ResponseError error 200.)")
2024-12-09T06:48:43.803Z V [Polar SDK] [BLE] ERROR: The operation couldn’t be completed. (ResponseError error 200.)
2024-12-09T06:48:43.803Z V [Polar SDK] [BLE] Error in writeFirmwareToDevice(): The operation couldn’t be completed. (ResponseError error 200.)
```

**How to Reproduce**
Call `updateFirmware` on iOS with Polar 360.

**Expected behavior**
Firmware update should not fail

**Screenshots and logs**

<details><summary>Details</summary>
<p>

![CleanShot 2024-12-09 at 15 17 34@2x](https://github.com/user-attachments/assets/82b2cb91-bb3e-4612-be69-327ac974431e)

```
2024-12-09T06:48:42.740Z V [Polar SDK] [BLE] waitDeviceSessionWithPftpToOpen(): isSessionOpen true, isFTPReady true
2024-12-09T06:48:42.740Z V [Polar SDK] [BLE] waitDeviceSessionWithPftpToOpen(): completed
2024-12-09T06:48:42.740Z V [Polar SDK] [BLE] Initialize session
2024-12-09T06:48:42.741Z V [Polar SDK] [BLE] PS-FTP new notification operation started for ID: 8
2024-12-09T06:48:42.741Z V [Polar SDK] [BLE] Notification packets counter reset
2024-12-09T06:48:42.741Z V [Polar SDK] [BLE] Transmitting notification packet, size: 2 bytes, for notification ID: 8
2024-12-09T06:48:42.741Z V [Polar SDK] H2D send  HEX 02 08
2024-12-09T06:48:42.741Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:42.741Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C53-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] Incrementing notificationPacketsWritten for PSFTP_H2D_NOTIFICATION_CHARACTERISTIC
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] PS-FTP notification sent successfully for ID: 8
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] PS-FTP Notification send operation DISPOSED for ID: 8
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] Closing totalStream in PS-FTP notification operation for ID: 8
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] PS-FTP new notification operation started for ID: 0
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] Notification packets counter reset
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] Transmitting notification packet, size: 2 bytes, for notification ID: 0
2024-12-09T06:48:43.006Z V [Polar SDK] H2D send  HEX 02 00
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.006Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.245Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C53-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.245Z V [Polar SDK] [BLE] Incrementing notificationPacketsWritten for PSFTP_H2D_NOTIFICATION_CHARACTERISTIC
2024-12-09T06:48:43.245Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.245Z V [Polar SDK] [BLE] PS-FTP notification sent successfully for ID: 0
2024-12-09T06:48:43.245Z V [Polar SDK] [BLE] PS-FTP Notification send operation DISPOSED for ID: 0
2024-12-09T06:48:43.245Z V [Polar SDK] [BLE] Closing totalStream in PS-FTP notification operation for ID: 0
2024-12-09T06:48:43.742Z V [Polar SDK] [BLE] Start /BTUPDAT.BIN write
2024-12-09T06:48:43.742Z V [Polar SDK] [BLE] PS-FTP new write operation
2024-12-09T06:48:43.742Z V [Polar SDK] [BLE] Transmitting first MTU packet, size: 495 bytes
2024-12-09T06:48:43.743Z V [Polar SDK] [BLE] waitDeviceSessionWithPftpToOpen(): isSessionOpen true, isFTPReady true
2024-12-09T06:48:43.743Z V [Polar SDK] [BLE] waitDeviceSessionWithPftpToOpen(): completed
2024-12-09T06:48:43.743Z V [Polar SDK] [BLE] Initialize session
2024-12-09T06:48:43.744Z I [PC] updateFirmware status: writingFwUpdatePackage(details: "Writing firmware update file BTUPDAT.BIN, bytes written: 476/175092 bytes")
2024-12-09T06:48:43.744Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C51-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.744Z V [Polar SDK] [BLE] Incrementing packetsWritten for PSFTP_MTU_CHARACTERISTIC
2024-12-09T06:48:43.744Z V [Polar SDK] [BLE] PS-FTP new notification operation started for ID: 8
2024-12-09T06:48:43.745Z V [Polar SDK] MTU send  HEX 06 10 00 08 01 12 0C 2F 42 54 55 50 44 41 54 2E 42 49 4E 3D B8 F3 96 00 00 00 00 00 02 00 00 10 A9 02 00 04 00 00 00 01 00 00 00 00 00 00 00 00 00 00 00 FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF
2024-12-09T06:48:43.745Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.745Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.745Z V [Polar SDK] [BLE] Notification packets counter reset
2024-12-09T06:48:43.745Z V [Polar SDK] [BLE] Transmitting notification packet, size: 2 bytes, for notification ID: 8
2024-12-09T06:48:43.745Z V [Polar SDK] [BLE][ERROR] Error processing interrupted frame: emptyQueueSignal
2024-12-09T06:48:43.745Z V [Polar SDK] [BLE] Writing firmware update file, bytes written: 476/175092
2024-12-09T06:48:43.746Z V [Polar SDK] [BLE] Writing firmware update file BTUPDAT.BIN, bytes written: 476/175092 bytes
2024-12-09T06:48:43.746Z V [Polar SDK] [BLE] Transmitting subsequent MTU packet, size: 495 bytes
2024-12-09T06:48:43.746Z V [Polar SDK] H2D send  HEX 02 08
2024-12-09T06:48:43.746Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.746Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.747Z V [Polar SDK] MTU send  HEX 17 FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF FF 8A F0 DA CF 16 BB FC B9 27 29 05 1D 41 F4 5F 2B BE 0E BA 30 A3 9C 68 C3 0C 47 84 90 44 29 47 F0 6E D7 99 6F 7E BF 8E FC D8 DD 86 12 B4 16 98 D4 CA E9 ED F4 EB 1A 84 DE C2 93 78 F9 58 20 09 E1 DC 08 9E F8 D0 79 24 0C 61 8D 26 07 1B 64 39 EB 60 D2 5D E1 F4 D1 8E F3 96 8A 57 A9 A1 54 8A 5F 98 F0 21 1F 3E 43 6C 9D 42 62 BB 07 94 F1 D1 C1 81 09 52 E4 58 10 8C 86 4F 9F 75 CB 68 47 D6 BF AC 35 71 55 5A E3 37 F5 85 0F 53 D7 2A A2 AE 2B 8A 53 EC 2F AE 7D 1C 25 C9 77 43 4F E9 19 8A 8A 14 5F 03 61 A9 B7 30 BD 32 1E 40 CD 94 27 46 A0 39 A6 DB 3E 43 F9 0B 2C 2C 70 AD 51 71 74 D9 A7 CF F5 B3 AD FB 0E 25 6D 8F D5 B0 0F F5 9A 01 E3 67 D1 87 E6 A8 07 95 52 54 DC 58 27 75 C7 9C 04 21 1B AA 6A 4E 30 C4 14 65 0B 27 A9 2D 98 E5 03 B4 F8 DE A2 91 CE 0B E2 4B 90 E2 84 D7 EB AA 43 32 50 AC 8C D4 C6 38 CD 83 95 CD B3 AA 30 F7 5C AB FC 2D AC 29 FC 72 7F F5 90 F5 35 3B 96 A4 3D 63 67 5A A3 8E 6B 44 D1 96 14 BF A3 75 12 12 44 0F 5E 88 83 5A EB 56 42 B2 D1 35 D5 F0 74 B9 35 EF D1 E2 9D 2C 02 97 F7 F3 F5 DC FD 4F 8C AC 41 18 AD 35 77 E6 35 AF 0E AF 68 E6 05 56 65 B2 09 FF F0 46 EF 92 12 6B 61 ED 3C D2 60 C8 4F 15 3D 00 47 86 62 4C 10 89 F6 DC D5 CF 65 41 4F 45 42 7A 5C 79 B0 6D C4 23 76 C3 2F 50 E6 A2 BA D6 DF 30 CB 0A 29 8F 24 7A E6 24 88 70 30 4B E8 2E 87 63 0B FD B9 25 3F 8A BD A3 12 44 39 46 A5 BD 5A E6 C0 E4 DE 47 1D FB A4 F1 91 33 12 7B CC 87 C4 B6 E4 3E 94 D0 26 90 14 A3 03
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] peripheralIsReadyToSendWriteWithoutResponse
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C51-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] Incrementing packetsWritten for PSFTP_MTU_CHARACTERISTIC
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE][ERROR] Error processing interrupted frame: emptyQueueSignal
2024-12-09T06:48:43.747Z V [Polar SDK] [BLE] Transmitting subsequent MTU packet, size: 495 bytes
2024-12-09T06:48:43.748Z V [Polar SDK] MTU send  HEX 27 9D 5E E4 3D A7 5B C5 DC 48 40 F6 05 A8 9A 43 0D 9C 63 DD CB 10 7E F0 21 DD B3 F6 D7 A0 E9 95 40 F4 62 39 C1 AF DA 30 58 3D 82 F9 02 1B 82 DF AC F8 FF C5 7C 5B ED C5 C3 20 19 0A 55 9A D3 83 C5 FA 0C 03 42 98 18 21 EE 49 7C 33 2C EA 50 58 6B 7E 85 30 70 57 B3 05 ED 5D 55 4D C5 E9 59 3A 42 C1 0C EB A3 EF 0C 23 4C C5 DE 41 5B 04 1F E3 24 9E 51 85 68 ED 77 EA 1E 8D 5A A5 FF 9C 86 B5 1A 94 4A 82 AC AF 8A 06 DC C5 98 9C CC 21 2E FE 38 8C 36 C8 61 2A 09 AC 75 A8 87 D3 F4 6C 7D 59 4D C8 44 A3 77 92 2D DF 91 EF 00 31 6E BB EA 90 D3 D9 74 EE A9 FD 78 CB 1A 34 3A 0E 8D 78 E3 AC 8D 0C 90 34 AD B4 4D 0B D0 F0 75 0E 14 4B 43 E1 26 E9 F9 6E 60 DD FA 8C F6 C7 04 4E 6C FC DD BE 08 64 0A 7C 47 93 3E C8 FE 45 47 75 9B FE 32 B5 08 16 5B 1E 4C 7D 2A 04 9F F7 B6 F4 F2 AA 05 75 A2 06 7D 40 D1 72 F0 F2 7A 95 D2 77 E0 A2 D7 FD 19 2D E7 C4 DA 5E 47 33 2F 1D 03 11 7B 28 FF 29 00 28 69 45 A4 22 22 79 CF B3 E6 FA F7 3C 61 DF D1 56 BD D9 F2 08 41 9C 5A 91 CD 10 8B AE F4 F2 8A 87 FA B6 C5 E9 D5 0E FA FF CB D3 F8 86 C9 1A 90 37 9D 9A 5F 26 DA 4A 8B 20 04 1A 61 D3 A5 B6 0C 88 D6 2D DC 98 96 1E 82 A7 7A 5F 12 4C 94 18 72 9F 3F 94 6F FB 21 F5 D4 8E FB B4 2A A2 0E 3D 17 63 73 46 95 41 FC C1 2A 89 6B CE D0 02 3B BF FE 1C 1E 3D 89 5A A9 4B F8 2F 0C C1 83 69 02 C1 E0 7F B7 84 43 3E 50 B4 76 4C 37 0A 82 D8 7D 64 BE D1 E7 14 1D B2 01 F1 C3 9C 00 6B E0 A4 37 B8 56 2E 87 6C 1C ED F4 4E 53 7D E2 0E 6C DB FE 6F E2 4F 21 B6 70 39 93 88 F8 EA FE 3F 73 12 69 9B 5A 3E 45 94 5C B0 B3 53 38 04 E9 ED 9F E3 C0
2024-12-09T06:48:43.748Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.748Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.768Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C51-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.768Z V [Polar SDK] [BLE] Incrementing packetsWritten for PSFTP_MTU_CHARACTERISTIC
2024-12-09T06:48:43.768Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.768Z V [Polar SDK] [BLE][ERROR] Error processing interrupted frame: emptyQueueSignal
2024-12-09T06:48:43.768Z V [Polar SDK] [BLE] Transmitting subsequent MTU packet, size: 495 bytes
2024-12-09T06:48:43.770Z V [Polar SDK] MTU send  HEX 37 0C 14 01 C4 3D 04 3C 93 B5 9E 43 5E 99 3E A6 76 D8 F1 CB 60 A6 DE 46 90 55 5B CF 8A F7 C9 15 A3 02 2F B2 C4 17 21 96 01 B3 7C 2B 92 8F C6 0E 37 3F DC B8 88 39 77 FF 7D 40 48 3C 22 B9 6D 08 1A EB 0E 5E 8A F7 BE 3B 0C 79 15 A4 BF 58 9B A6 D7 F1 C0 77 7D 71 3A BD 30 F3 B7 21 86 D1 AA 5A 99 90 6B 73 40 DF 25 B6 8F E7 2D 60 FB 4D 06 D4 1A 02 F6 BC 3B AE 6C 9A 7D 14 CA B6 C5 F1 3E D8 56 72 10 09 98 C7 08 18 43 63 F0 95 92 7A D6 D5 93 F6 AF F1 45 9C E9 50 DE 71 87 32 A8 33 D6 CF E8 E4 C1 75 05 E4 47 7B DD E1 50 64 35 C0 F7 1C 09 B3 3E 58 5B 49 5A ED 9D B8 E9 15 6F 27 DC 1E 74 53 BE F9 90 4A D6 8B 1C 66 B0 73 77 A2 45 E8 A2 54 0D DC 11 EA 65 C9 D1 76 95 47 5B 9C 44 33 86 90 D3 DC 10 44 24 09 0E 2A 70 17 6D EE FC 89 77 99 1D 56 8A C7 2F EF C8 A9 A1 CF 3C 86 A7 E6 FE 47 41 C5 CF 84 13 16 8D DC ED E5 3B 3E 56 C4 4A B3 29 18 1E 83 03 A5 0B 3B C1 6D D9 2C E9 03 EE 27 E3 B5 99 95 D1 CA 52 97 EC 1D 6C 74 2F 1F A9 7D 41 7B 89 CE A3 F6 7B 90 59 2A 78 9A 14 F6 42 F3 D4 86 76 26 01 16 8A D3 BE 86 F7 16 41 54 4D 16 5A 34 FF 4A 4A 59 3A 65 05 CA 3B FE F9 8E AD 59 36 0D A5 65 63 63 41 4A AC 5A 8A 5A 10 45 B8 FD 1C 63 A1 43 9E D4 1A C4 79 3C CC 9D 12 23 DB 80 0E 58 17 C0 7E 83 EF D5 C6 B9 03 E5 FD 3E 73 CE E7 2E 6D B6 D1 CF 29 9E 0B 76 6B 52 FD D8 FC 9A 0E 54 2A C1 75 06 47 8A FD 97 C7 03 B5 30 C0 F8 43 2E 25 8E 86 8C D3 C4 C9 75 90 51 2D F0 1C 27 15 14 9F DC 87 A7 BA F1 19 65 8F 0D 84 75 E6 80 19 DA DF 25 13 7E 75 E2 4D 0F 22 51 03 FF E6 D9 97 BC 1A CF D3 C7 20 76 34 2F 2C 50 75
2024-12-09T06:48:43.770Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.770Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.789Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C51-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.789Z V [Polar SDK] [BLE] Incrementing packetsWritten for PSFTP_MTU_CHARACTERISTIC
2024-12-09T06:48:43.789Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.789Z V [Polar SDK] [BLE][ERROR] Error processing interrupted frame: emptyQueueSignal
2024-12-09T06:48:43.789Z V [Polar SDK] [BLE] Transmitting subsequent MTU packet, size: 495 bytes
2024-12-09T06:48:43.790Z V [Polar SDK] MTU send  HEX 47 D1 AD 7E 60 0B 2B DF 67 3F 76 81 BD 9E 3B DD 5B 8A ED C3 8C 09 F3 A6 6C EA C6 39 73 8C 7B 11 F0 B0 4A DB 98 02 2C 9B F8 52 8C 86 5B A2 F7 18 42 15 4F 24 7C 33 56 09 9C DE DE 79 A2 58 60 A1 CB E3 F6 B6 90 03 CF 7D FA 87 5C 09 EA E7 5B 75 10 7A 4B 8A 59 A8 27 A4 DD 74 4A 51 3F 15 00 8E 30 F2 00 1A F2 DB 20 C9 AE 5B E0 CC EF C1 78 2F 70 3D 40 6A 0C 31 76 B1 DB C1 21 D4 E3 63 2C 3D E8 3A FF A0 E2 7D 30 B9 0E 4A DD 9A 3D 94 CB 1F B1 6D 2F 37 63 5C F1 62 2D CF E2 F7 29 4D 94 3B 26 98 49 D3 3B E4 CE 02 09 9F DC 0E F3 6F 25 CB 00 D2 66 1D A2 E9 D8 96 DB 48 C0 78 73 74 FF 51 27 1E 72 F3 E1 37 E9 7F 54 86 79 CE 38 4B 4D 87 8C AD A1 17 56 A9 B5 95 46 9D 5A 18 14 54 C3 AB 5C 7F 73 72 4E E8 17 D5 63 A9 A9 D3 92 C4 7C F0 F8 1A E1 05 57 4D 20 70 58 4B B1 49 EE 41 92 58 8E 7F AB F4 F5 DF FE 3A 5E F2 CD 25 FE EF BC C8 7E A6 8B 7C 98 C7 7E FB 5D 7E E7 1A 88 F7 D1 4C 9F CE 5E E0 04 BA 5D 83 C3 E5 E5 48 3D 9C DA 27 DB B6 1D F0 A6 6B F2 CC C7 D6 AE 88 BC B9 FD D6 6A DC E2 05 AE 73 30 DF 06 E4 B7 BA C2 12 5E 54 B5 C2 B0 58 92 7D 39 31 76 9F 86 FE 57 34 97 28 DA D3 1D 60 8B 70 0F 45 34 0E 0C 2C 02 FB 6D 48 28 FB D9 5E 37 6B 0E A2 4B AB 0B AB 0A 21 6B F3 7D 79 ED 3B 47 2B EC D4 71 DC D9 3F C4 65 76 25 F3 FF AE AC 78 42 F1 44 22 F3 CE C5 D2 8B 27 88 59 5F EA 4A 75 88 52 70 8F A8 C9 00 2D ED D6 8D 5A 02 72 E3 83 98 3C CB 9F A1 2F 70 5D D1 EB FD BF ED 41 DF 04 8C 96 A8 B8 88 C3 34 2B 22 2C 68 3E 75 93 B2 B8 33 78 7E B0 F1 49 0B A3 32 01 DD 4F 73 17 AE 84 DA DC FE 53 88 B3 9B 31 3F 62
2024-12-09T06:48:43.790Z V [Polar SDK] [BLE] waitPacketsWritten started with count: 1, timeout: 90.0 seconds
2024-12-09T06:48:43.790Z V [Polar SDK] [BLE] Current written: 0, Target count: 1
2024-12-09T06:48:43.801Z V [Polar SDK] [BLE] processServiceData called with characteristic: FB005C51-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0, data size: 3 bytes
2024-12-09T06:48:43.801Z V [Polar SDK] Processing PSFTP_MTU_CHARACTERISTIC  HEX 00 C8 00
2024-12-09T06:48:43.802Z V [Polar SDK] [BLE] All packets written successfully, resetting written count
2024-12-09T06:48:43.802Z V [Polar SDK] [BLE] Frame sending interrupted by device!
2024-12-09T06:48:43.802Z V [Polar SDK] [BLE] serviceDataWritten called with characteristic: FB005C53-02E7-F387-1CAD-8ACD2D8DF0C8, error: 0
2024-12-09T06:48:43.802Z V [Polar SDK] [BLE] Incrementing notificationPacketsWritten for PSFTP_H2D_NOTIFICATION_CHARACTERISTIC
2024-12-09T06:48:43.802Z V [Polar SDK] [BLE] peripheralIsReadyToSendWriteWithoutResponse
2024-12-09T06:48:43.803Z I [PC] updateFirmware status: fwUpdateFailed(details: "Error during firmware update: The operation couldn’t be completed. (ResponseError error 200.)")
2024-12-09T06:48:43.803Z V [Polar SDK] [BLE] ERROR: The operation couldn’t be completed. (ResponseError error 200.)
2024-12-09T06:48:43.803Z V [Polar SDK] [BLE] Error in writeFirmwareToDevice(): The operation couldn’t be completed. (ResponseError error 200.)
2024-12-09T06:48:43.803Z V [Polar SDK] [BLE][ERROR] Error during firmware update: responseError(errorCode: 200)
```

</p>
</details>

**Comment by samulimaa** [comment]
This should be fixed in the latest SDK release 5.10.0.

**Comment by pth-pe-gh** [comment]
FLOW-60986
